src/solc_0.8/common/Libraries/BytesUtil.sol

```diff
@@ -7,8 +7,8 @@ library BytesUtil {
     /// @param _address The address to compare to.
     /// @return Whether the first param == _address.
     function doFirstParamEqualsAddress(bytes memory data, address _address) internal pure returns (bool) {
-        if (data.length < 68) {
-            // 68 = 36 + 32
+        uint256 dataMaxLength = 68; // 68 = 36 + 32
```

Comment:
I was thinking about a uint256 private constant for 36 and 32. Where does this number come from?
something like   if (data.length < MEANING_36 + MEANING_32) { ...

Reply:
I tried to understand that, but could just yet. So at this point I am not quite sure what values 36 and 32 represent. I just wanted to avoid the calculation every time this function is called.
And well yeah I should have made it a constant :P